### DISCUSSION TITLE
Data acquisition task

### DISCUSSION THREAD

**Discussion by xifen523** [abizovnuralem/go2_ros2_sdk#42] [Q&A] [answered]
Thank you for this exciting work. Since the EDU version is too expensive to afford, see that you develop an SDK that can be supported in a non-EDU version.

I would like to know if your SDK can support data acquisition? I want to use GO2 to capture data for AI training, for example, I want to get raw point cloud and image data, preferably all sensor data. Then the time between the data can be synchronized, can this be achieved?

**Comment by abizovnuralem** [comment, marked as answer]
You can get all the necessary data from the dog to train your AI. Point Cloud etc are available from a webrtc protocol.
